specreduce/background.py

```diff
@@ -303,12 +303,18 @@ def bkg_image(self, image=None):
 
         Returns
         -------
-        `~specutils.Spectrum1D` object with same shape as ``image``.
+        spec : `~specutils.Spectrum1D`
```

Comment:
The Sphinx ref will have to be updated separately after specutils 2.0 stable is released and RTD picks up specutils 2.0. Might have to do a special requirements file for doc build or pin `specutils>=2` here:

https://github.com/astropy/specreduce/blob/6a0b915e0dc131de0f19a7c2b417d25ab0e27bb2/pyproject.toml#L25

But nothing we can do right now.

Reply:
ah, ignore my other comment, then.